media/static-bundles.json

```diff
@@ -311,7 +312,8 @@
     {
       "files": [
         "css/pebbles/global.scss",
-        "css/pebbles/protocol-nav.scss"
+        "css/pebbles/protocol-nav.scss",
+        "css/pebbles/protocol-footer.scss"
```

Comment:
Can we now remove the `components/footer` import from `global.scss` if all locales are getting the protocol footer? :)

Reply:
We can likely also remove all the `#colophon {}` styles from Sandstone's monolithic stylesheet?

Reply:
Yes. Yes we can!

Reply:
✂️